index.html

```diff
@@ -8,10 +8,30 @@
     <title>Portfolio</title>
 </head>
 <body>
-    <nav>
-        <ul>
-            <li>Logo</li>
-        </ul>
-    </nav>
+    <div id="header-background">
+        <nav id="flex-nav">
+            <ul>
+                <li id="logo"><a href="index.html">My Logo</a></li>
+                <li id="burguer-menu-icon ">
+                    <img src="img/menu.svg" alt="burguer-menu" class="burguer-menu-vector">
+                </li>
+            </ul>
+        </nav>
+        <div id="primary-text">
+            <h2 id="primary-text-header">I'm Enoque <br/> Glad to see<br/>you!</h2>
```

Comment:
Since this is the main title of this section, it would be better to use an `h1` tag. This helps in improving your website to increase its visibility when people search for products or services (SEO optimization).
You can even break it into parts so that you use `h1` tag for `I'm Enoque` and `h2` tag for `Glad to see you`